debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -262,9 +262,10 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
             .withWidth(Width.SHORT)
             .withImportance(Importance.LOW)
             .withDescription("The criteria for running a snapshot upon startup of the connector. "
-                    + "Options include: "
-                    + "'initial' (the default) to specify the connector should run a snapshot only when no offsets are available for the logical server name; "
-                    + "'schema_only' to specify the connector should run a snapshot of the schema when no offsets are available for the logical server name. ");
+                    + "Select one of the following snapshot options: "
+                    + "'initial' (default): If the connector does not detect any offsets for the logical server name, it runs a snapshot that captures the current full state of the configured tables. After the snapshot completes, the connectors begins to stream changes from the transaction log.; "
+                    + "'initial_only': The connector performs a snapshot as it does for the 'initial' option, but after completing the snapshot, it does not stream changes from the transaction log.; "
+                    + "'schema_only': If the connector does not detect any offsets for the logical server name, it runs a snapshot that captures only the schema (table structures), but not any table data.");
```

Comment:
If we mention that for `initial`, the connector begins to stream changes from the transaction log, the same should be appended to the `schema_only` option for consistency and clarity.  I also believe we should mention that the connector stops after the snapshot finishes for `initial_only`.